internal/commands/app/create_inputs_test.go

```diff
@@ -710,90 +710,166 @@ func TestAppCreateInputsResolveCluster(t *testing.T) {
 		assert.Equal(t, "123", expectedGroupID)
 	})
 
-	t.Run("should error when more than one cluster passed in when creating template app", func(t *testing.T) {
-		_, ui := mock.NewUI()
-		clusterNames := []string{"Cluster0", "Cluster1"}
+	t.Run("when creating template apps", func(t *testing.T) {
```

Comment:
i decided to group all of the template app creation tests together to tidy them up a bit

Reply:
hiding whitespace changes will make reading through this significantly easier